viewer/src/main/webapp/viewer-html/common/viewercontroller/ViewerController.js

```diff
@@ -926,7 +926,18 @@ Ext.define("viewer.viewercontroller.ViewerController", {
                     transparent: true,
                     noCache: true
                 };
-                
+                if(service.version === "1.3.0"){
+                    ogcOptions={
```

Comment:
you only need the overrides here (I think only `version` and `crs`) and srs should then be deleted